app/src/main/java/com/nitramite/paketinseuranta/Constants.java

```diff
@@ -21,6 +21,7 @@
     final static String SP_MAIN_MENU_LIST_LONG_TAP_ACTION = "SP_MAIN_MENU_LIST_LONG_TAP_ACTION";
     final static String SP_UPDATE_FAILED_FIRST = "SP_UPDATE_FAILED_FIRST";
     public final static String SP_THEME_SELECTION = "SP_THEME_SELECTION";
+    public final static String SP_PACKAGE_LAST_CHANGE = "SP_PACKAGE_LAST_CHANGE";
```

Comment:
Rename to `SP_SHOW_LAST_EVENT_DATE` for consistency and to make it clear this is a setting toggle (if I understood correctly).